src/test/java/entities/HumanPlayerTest.java

```diff
@@ -0,0 +1,72 @@
+package entities;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HumanPlayerTest {
+
+    HumanPlayer p1;
```

Comment:
Add access modifiers to class attributes and methods

Reply:
Fixed for all test classes.